Src/Innovative.Blazor.Components/Components/Form/InnovativeForm.razor.css

```
@@ -187,12 +187,52 @@
 ::deep .rz-chkbox-box:hover:not(.rz-state-disabled):after {
     width: var(--inn-input-checkbox-circle-hover-width);
     height: var(--inn-input-checkbox-circle-hover-height);
     background-color: var(--inn-input-checkbox-circle-hover-background-color);
 }
 
+::deep .rz-chkbox-box.rz-state-active {
+    background-color: var(--inn-primary-lighter);
+    border: 1px solid var(--inn-primary);
+}
+
+::deep .rz-chkbox-box.rz-state-active:hover:not(.rz-state-disabled) {
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-elements: deep

See MDN web docs for more details.
Use a known pseudo-elements instead, such as:


(lint/correctness/noUnknownPseudoElement)